opentelemetry-api/src/opentelemetry/propagate/__init__.py

```diff
@@ -99,26 +99,26 @@ def extract(
         context: an optional Context to use. Defaults to current
             context if not set.
     """
-    return get_global_textmap().extract(getter, carrier, context)
+    return get_global_textmap().extract(carrier, context, getter=getter)
 
 
 def inject(
-    set_in_carrier: textmap.Setter[textmap.TextMapPropagatorT],
-    carrier: textmap.TextMapPropagatorT,
+    carrier: textmap.CarrierT,
     context: typing.Optional[Context] = None,
+    setter: textmap.Setter = textmap.default_setter,
 ) -> None:
     """Uses the configured propagator to inject a Context into the carrier.
 
     Args:
-        set_in_carrier: A setter function that can set values
+        setter: A `Setter` object that can set values
```

Comment:
This should be moved to below the context, ditto with the getter documentation above.


```suggestion
        setter: An optional `Setter` object that can set values
```